commerce-test/src/testIntegration/java/com/liferay/commerce/order/stock/test/OrderStockManagementTest.java

```diff
@@ -84,12 +105,15 @@ public void testBackOrder() throws Exception {
 		);
 
 		CommerceCurrency commerceCurrency =
-			CommerceCurrencyTestUtil.addCommerceCurrency(_group.getGroupId());
+			CommerceCurrencyTestUtil.addCommerceCurrency(_company.getGroupId());
```

Comment:
@riccardo-alberti  @ethanbustad  CommerceCurrencyTestUtil doesn't need anymore a groupId, we can refactor a delete this parameter as the Currencies are scoped to the company now.